test/functional/specs/desktop/C2583.js

```diff
@@ -12,74 +12,34 @@ test.meta({
 
 test("Regression: Set the log option to true. Load the page. Execute an event command.", async () => {
   await t
-    .click(
-      Selector(
-        "#body > section > div.left-nav > div > ul > li:nth-child(1) > a"
-      )
-    )
-    .click(
-      Selector(
-        "#body > section > div.left-nav > div > ul > li:nth-child(4) > a"
-      )
-    );
+    .click(Selector("#logenabled-button"))
+    .click(Selector("#event-button"));
 
   const message = await t.getBrowserConsoleMessages();
 
   await t.expect(message.log).match(/\[alloy] Executing event command./);
 
-  // Remove the log option from the configuration. Refresh the browser. Execute an event command.
   await t
-    .click(
-      Selector(
-        "#body > section > div.left-nav > div > ul > li:nth-child(2) > a"
-      )
-    )
-    .click(
-      Selector(
-        "#body > section > div.left-nav > div > ul > li:nth-child(4) > a"
-      )
-    );
+    .click(Selector("#nologconfig-button"))
+    .click(Selector("#event-button"));
 
   const message2 = await t.getBrowserConsoleMessages();
 
-  await t
-    .expect(message2.log)
-    .match(/Config nologconfig initiated./)
-    .expect(message2.log)
-    .match(/Config event initiated./)
-    .expect(message2.log)
-    .match(/\[alloy] Executing event command./);
+  await t.expect(message2.log).match(/\[alloy] Executing event command./);
```

Comment:
FYI, this behavior will change in the future: https://jira.corp.adobe.com/browse/CORE-33754